### ISSUE TITLE
Investigate Litmus chaos framework

### ISSUE THREAD

**Issue by chaitanyaenr** [openshift-scale/kraken#31]
CNV team looked into it and found out that Litmus framework ( https://github.com/litmuschaos/litmus ) has interesting chaos scenarios that Kraken is missing today. We need to investigate to see if it makes sense to consume it under the hood to improve the coverage.

**Comment by paigerube14** [comment]
So far I have not been able to run the pod-delete scenario given in the start up guide. I tried adding an annotation to 'openshift-apiserver" deployment and edited the chaos engine yaml to point to this deployment. I keep getting the error that "Verify that the AUT (Application Under Test) is running (pre-chaos)" even though it is running and I can see it has 3 running pods. Need to do some further investigation
I was able to get a couple of the infra chaos experiments related specifically to nodes, to work properly. I didn't have any luck using the yaml file examples in the litmus docs. I used the 3 yaml files in the hub.litmus (chaos experiment, rbac and chaos engine) to properly run the experiment which was a little different from what the docs had. Not sure if this is 100% the right way to run but that was what I found worked. 

Seeming like if we can install litmus and make sure the components are up at the beginning of kraken running we should be able to apply the yaml files for each scenario and check the status (this is the part that could take a while) Should be pretty straight forward
